docs/use/RestoreBackup.md

```diff
@@ -1,16 +1,28 @@
-
 # Restore the database from a previously saved backup
 
+Database backups and data replication are complementary components of an effective disaster recovery strategy. 
+
+Everest enables you to use backups to either create a new database or restore the backup data to the original database. Choosing between these two options often depends on the specific scenario, the urgency of recovery, the impact on ongoing operations, the need for data isolation, and the purpose of the restoration (such as testing, recovery from corruption, or analysis).
+
+## Restore database from backup
+
+Restoring to the same cluster is useful in the following scenarios:
+
+- **When restoring from a Point-in-time recovery (PITR) timestamp** as PITR does not currently support restoring to a new database.
```

Comment:
This isn't a pro of restoring to the same DB instead of creating a new one, it's just our current limitation. I wouldn't mention this as a pro of restoring to the same DB

Reply:
hmm, the bullets just lists are introduced as specific scenarios when it makes sense to use the restore type, not as pro/cons..

" Restoring to the same cluster is useful in the following scenarios"
" There are specific situations that require the creation of a new database from a backup instead of restoring the data to the original database"

Reply:
Ok, that's fair, having a second look at it, it looks good :slightly_smiling_face: